src/components/video-view.tsx

```diff
@@ -139,8 +165,15 @@ export const VideoView = ({
         timelineMarks={timelineMarks}
       />
       <div className={clsx("video-pane", ac.mode, extraClass)}>
+        <video
+          ref={backVideoRef}
+          className={clsx("video-view-video", extraClass)}
+        >
+          <source src={backVideoFile || aniVideos.tissue.heart[0][0]} type={"video/mp4"} />
```

Comment:
Rather than repeating yourself (here and below), you could define a constant:
```
const kDefaultVideo = aniVideos.tissue.heart[0][0];
```
and then
```suggestion
          <source src={backVideoFile || kDefaultVideo} type={"video/mp4"} />
```